bedrock/firefox/views.py

```diff
@@ -715,7 +715,7 @@ def new(request):
             elif experience == 'auf-deiner-seite':
                 template = 'firefox/new/berlin/scene1-auf-deiner-seite.html'
             else:
-                template = 'firefox/new/scene1.html'
+                template = 'firefox/new/scene1.de.html'
```

Comment:
Locale specific templates work automatically, so you shouldn't need to specify anything in the view here for them to work.